README.md

```diff
@@ -8,7 +8,7 @@ Examples for how to use the Flink Docker images in a variety of ways.
 Docker Compose
 --------------
 
-Use the [Docker Compose config](docker-compose.yml) in this repo to create a local Flink cluster.
+Use the [Docker Compose config](docker/docker-compose.yml) in this repo to create a local Flink cluster.
```

Comment:
Unfortunately there is a reference to this file in [Flink's docs](https://github.com/apache/flink/blob/release-1.3/docs/setup/docker.md#flink-with-docker-compose) so we need to update that and probably maintain a symlink here until the next release.